Source/WSTagsField.swift

```diff
@@ -66,6 +66,12 @@ open class WSTagsField: UIScrollView {
             tagViews.forEach { $0.displayDelimiter = self.isDelimiterVisible ? self.delimiter : "" }
         }
     }
+    
+    open var isImageVisible: Bool = false {
```

Comment:
From what I can tel, this  sets all the tag's to show their images, correct? Perhaps a name like `showImages` might be clearer (or something like that)? `isImageVisible` makes it feel like we are checking / setting a single image.